### DISCUSSION TITLE
Evaluate Windows Admin Center

### DISCUSSION THREAD

**Discussion by jemrobinson** [alan-turing-institute/data-safe-haven#1247] [Ideas]
Could [Windows Admin Center](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/overview) be a replacement for our SHM VPN? It sounds like you can configure it to require MFA and it would be accessible via a URL and therefore (relatively) platform agnostic.

**Comment by martintoreilly** [comment]
Ooh. That looks 😎 For some (many?) parts it's quite possibly a replacement. Looks like it's "Server manager as a service". I think the things it doesn't support is connecting into any of the Linux VMs or running powershell that needs to be run on an actual VM (e.g. if we were to move to Azure AD Directory Services, I think some elements require running powershell modules only available on Windows).

**Comment by martintoreilly** [comment]
> I think the things it doesn't support is connecting into any of the Linux VMs or running powershell that needs to be run on an actual VM

Though maybe there is a way to [run powershell in the admin centre](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/use/get-started#view-powershell-scripts-used-in-windows-admin-center)
